src/normal_form_game.jl

```diff
@@ -53,6 +53,42 @@ function Base.show(io::IO, player::Player)
     Base.print_array(io, player.payoff_array)
 end
 
+# delete_action
+
+"""
+    delete_action(player, action, player_idx)
+
+Return a new Player instance with the actions specified by
+`action` deleted from the action set of the player specified by
+`player_idx`.
+
+# Arguments
+
+- `player::Player` : Player instance.
+- `action::AbstractVector{<:PureAction}`: the actions to be deleted.
+- `player_idx::Integer` : Index of the player to delete actions for.
+
+# Returns
+
+- `::Player` : `Player` instance with the actions deleted as specified.
+
+#Reference
+
+https://stackoverflow.com/questions/53637511/how-to-delete-the-specific-row-of-n-dimensional-array-in-julia
```

Comment:
Do not put this in the docstring (this is just for internal implementation).

Just make a comment outside the docstring.